lib/grpperm.gi

```diff
@@ -2134,34 +2134,37 @@ local dom,s,cs,t,ts,o;
   elif Length(dom)=Size(s) then
     return "5";
   else
-    # now get one simple factor of the socle
+    # so the group now is of type 3 or 4. Next we determine a simple socle
+    # factor and a direct product of all but one socle factor.
+    # simple socle factor.
+    stb:=Stabilizer(s,1);
```

Comment:
Isn't this wrong if the socle's domain doesn't include `1`?

Reply:
```
gap> G := Group(GeneratorsOfGroup(PrimitiveGroup(3600,1)));;
gap> ONanScottType(G^(1,3601));
"4b"

Reply:
Yes. Thank you. Fixed.